i18n/en-US/discriminator.json

```diff
@@ -0,0 +1,6 @@
+{
+  "DESCRIPTION": "Provides a list of all uers with a given discriminator.",
+  "USAGE": "discriminator [four-digit-discriminator]",
```

Comment:
Can we change `[four-digit-discriminator]` to either `[discrim]` or `[discriminator]`

Reply:
I chose `[discriminator]` but what would you think about `[####]`